src/styles/theme/margin.scss

```diff
@@ -0,0 +1,16 @@
+$margin-sizes: (
+  "xsmall": $xsmall-margin,
+);
+$margin-directions: "top", "right", "bottom", "left";
+
+@each $name, $size in $margin-sizes {
+  .oc-margin-#{$name} {
```

Comment:
It's rather confusing mixing `uk-margin` with an additional `oc-margin` … I'd rather have it be an uk-margin-… instead.